shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -68,6 +68,58 @@ - (instancetype)initWithConnection:(NSNumber*)connection
 
 #pragma mark -
 
+// The information of a view that has been attached to the engine.
+@interface FlutterEngineViewRecord : NSObject
```

Comment:
Could you explain the benefits of adding this new `FlutterEngineViewRecord` interface? Alternatively, the `FlutterViewController` could own the bridge and track whether semantics is enabled.

Reply:
That's a good idea, and it saves me from writing alternatives to the `accessibilityBridge` getter.